syntax = "proto3";

package echo;

service EchoServer {
  rpc SayHello (EchoRequest) returns (EchoReply) {}
  rpc SayHelloStream(EchoRequest) returns (stream EchoReply) {}
}

message EchoRequest {
  string firstname = 1;
  string lastname = 2;
}

message EchoReply {
  string message = 1;
}
